akka-docs-new/src/main/paradox/guide/modules.md

```diff
@@ -0,0 +1,165 @@
+comes to concurrency and remote communication. Don’t worry if this description feels too high level to fully grasp 
+yet, in the next chapter we will explain actors in detail. For now, the important point that this is a model that 
+handles concurrency and distribution at the fundamental level instead of ad hoc patched attempts to bring these 
+features to OOP.
+
+Problems that actors solve include:
+
+* How to build and design high-performance, concurrent applications?
+* How to handle errors in a multi-threaded environment?
+* How to protect my project from the pitfalls of concurrency?
+
+### Remoting
+
+Remoting enables actors that are remote from each other, living on different computers, to seamlessly exchange messages. 
+While distributed as a JAR artifact, Remoting resembles a module more than it does a library. You enable it mostly 
+with configuration, it has only a few APIs. Thanks to the actor model, remote and local message sends look exactly the 
+same. The patterns that you use locally on multi-core systems translate directly to remote systems. 
+You will rarely need to use Remoting directly, but it provides the foundation on which the Cluster subsystem is built.
+ 
+Some of the problems Remoting solves are
+
+* How to address actor systems living on remote hosts?
+* How to address individual actors on remote actor systems?
+* How to turn messages to bytes on the wire?
+* How to manage low-level, network connections (and reconnections) between hosts, detect crashed actor systems and hosts, 
+  all transparently?
+* How to multiplex communications from unrelated set of actors on the same network connection, all transparently?
+
+### Cluster
+
+If you have a set of actor systems that cooperate to solve some business problem, then you likely want to manage these set of 
+systems in a disciplined way. While Remoting solves the problem of addressing and communicating with components of 
+remote systems, Clustering gives you the ability to organize these into a "meta-system" tied together by a membership
+protocol. **In most of the cases, you want to use the Cluster module instead of using Remoting directly.** 
+Cluster provides an additional set of services on top of Remoting that most of the real world applications need. 
+
+The problems the Cluster module solves (among others) are
+
+* How to maintain a set of actor systems (a cluster) that can communicate with each other and consider each other as part of the cluster?
+* How to introduce a new system safely to the set of already existing members?
+* How to detect reliably systems that are temporarily unreachable?
+* How to remove failed hosts/systems (or scale down the system) so that all remaining members agree on the remaining subset of the cluster?
+* How to distribute computations among the current set of members?
+* How do I designate members of the cluster to a certain role, in other words to provide certain services and not others?
+
+### Persistence
```

Comment:
move Peristence after the cluster sections, to keep everything cluster together?